kubemarine/templates/plugins/calico-v3.22.yaml.j2

```diff
@@ -4104,7 +4104,7 @@ spec:
         cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
     spec:
       nodeSelector:
-        {{ plugins['calico']['typha']['nodeSelector'] | toyaml | indent(width=8, indentfirst=False) }}
+        {{ plugins['calico']['typha']['nodeSelector'] | toyaml | indent(width=8, first=False) }}
```

Comment:
We no longer use templates for calico at all. I am not sure that they should be deleted as part of this PR, but anyway they should be either all or none deleted

Reply:
delete old file